mPower2/mPower2/View Controllers/ReminderTableStepViewController.swift

```diff
@@ -0,0 +1,124 @@
+// this list of conditions and the following disclaimer in the documentation and/or
+// other materials provided with the distribution.
+//
+// 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+// may be used to endorse or promote products derived from this software without
+// specific prior written permission. No license is granted to the trademarks of
+// the copyright holders even if such marks are included in this software.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+import UIKit
+import BridgeApp
+
+class ReminderTableStepViewController: RSDTableStepViewController {
+    
+    override func registerReuseIdentifierIfNeeded(_ reuseIdentifier: String) {
+
+        // We want to use the custom 'ReminderTextFieldCell' text field cell for the .picker type
+        if let _ = RSDTableItem.ReuseIdentifier(rawValue: reuseIdentifier) {
+            super.registerReuseIdentifierIfNeeded(reuseIdentifier)
+        }
+        else {
+            let reuseId = RSDFormUIHint(rawValue: reuseIdentifier)
+            switch reuseId {
+            case .picker:
+                let cellClass: AnyClass = ReminderTextFieldCell.self
+                tableView.register(cellClass, forCellReuseIdentifier: reuseIdentifier)
+            default:
+                super.registerReuseIdentifierIfNeeded(reuseIdentifier)
+            }
+        }
+    }
+
+    override func configure(cell: UITableViewCell, in tableView: UITableView, at indexPath: IndexPath) {
+        
+        super.configure(cell: cell, in: tableView, at: indexPath)
+        
+        // We don't want to show a label above the text field
+        if let textFieldCell = cell as? RSDStepTextFieldCell {
+            textFieldCell.fieldLabel.text = nil
+        }
+    }
+    
+    override func setupModel() {
+        
+        super.setupModel()
+        
+        // We want a section label above the time picker cell, which doesn't have one by default.
+        // So we set the title of that table item
+        (tableData?.sections.first)?.title = "Set Reminder"
+        
+        // We don't want a section label above the checkbox, so we nil the title of that table item
+        (tableData?.sections.last)?.title = nil
```

Comment:
FYI: This was a bug that has since been fixed. If there is a single check-box, it shouldn't include a title by default.